[auto-suggestion]Empty content prompts NPE

## Reproduce
In the console, when the page is empty, use the key combination option+space (mac) or control+space (Windows), NPE will occur, exception stack:

```
22/02/11 01:55:20  INFO SessionCleaner: clean org.apache.spark.sql.SparkSession@4110ffe5 List(org.apache.spark.sql.util.ExecutionListenerBus@250377fa)
java.lang.AssertionError: assertion failed: lineNum and columnNum should be set
	at scala.Predef$.assert(Predef.scala:223)
	at tech.mlsql.autosuggest.AutoSuggestContext._suggest(AutoSuggestContext.scala:168)
	at tech.mlsql.autosuggest.AutoSuggestContext.suggest(AutoSuggestContext.scala:160)
	at tech.mlsql.autosuggest.app.AutoSuggestController.run(MLSQLAutoSuggestApp.scala:114)
	at streaming.rest.RestController.script(RestController.scala:221)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.csdn.modules.http.RestController.filter(RestController.java:139)
	at net.csdn.modules.http.RestController.dispatchRequest(RestController.java:99)
	at net.csdn.modules.http.HttpServer$DefaultHandler.handle(HttpServer.java:182)
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:52)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
	at org.eclipse.jetty.server.Server.handle(Server.java:499)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:311)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)
	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:544)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:748)
22/02/11 01:55:20  ERROR http: System processing error
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.csdn.modules.http.RestController.filter(RestController.java:139)
	at net.csdn.modules.http.RestController.dispatchRequest(RestController.java:99)
	at net.csdn.modules.http.HttpServer$DefaultHandler.handle(HttpServer.java:182)
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:52)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
	at org.eclipse.jetty.server.Server.handle(Server.java:499)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:311)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)
	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:544)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: assertion failed: lineNum and columnNum should be set
	at scala.Predef$.assert(Predef.scala:223)
	at tech.mlsql.autosuggest.AutoSuggestContext._suggest(AutoSuggestContext.scala:168)
	at tech.mlsql.autosuggest.AutoSuggestContext.suggest(AutoSuggestContext.scala:160)
	at tech.mlsql.autosuggest.app.AutoSuggestController.run(MLSQLAutoSuggestApp.scala:114)
	at streaming.rest.RestController.script(RestController.scala:221)
	... 16 more
22/02/11 01:55:20  INFO impl: Completed 500 in 244ms 	POST /run/script



22/02/11 01:55:20  WARN HttpChannel: /run/script
java.lang.NullPointerException
	at java.io.PrintWriter.write(PrintWriter.java:473)
	at net.csdn.modules.http.DefaultResponse.output(DefaultResponse.java:196)
	at net.csdn.modules.http.DefaultResponse.error(DefaultResponse.java:190)
	at net.csdn.modules.http.HttpServer$DefaultHandler.defaultErrorAction(HttpServer.java:157)
	at net.csdn.modules.http.HttpServer$DefaultHandler.handle(HttpServer.java:194)
	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:52)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
	at org.eclipse.jetty.server.Server.handle(Server.java:499)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:311)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)
	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:544)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:748)
```
![image](https://user-images.githubusercontent.com/20615623/153467819-592923ef-d5c5-4757-b0da-40452f812438.png)

--- a/external/mlsql-autosuggest/src/main/java/tech/mlsql/autosuggest/AutoSuggestContext.scala
+++ b/external/mlsql-autosuggest/src/main/java/tech/mlsql/autosuggest/AutoSuggestContext.scala
@@ -165,7 +165,7 @@ class AutoSuggestContext(val session: SparkSession,
    * We need to convert it to the relative pos in every statement
    */
   private[autosuggest] def _suggest(tokenPos: TokenPos): List[SuggestItem] = {
-    assert(_rawColumnNum != 0 || _rawColumnNum != 0, "lineNum and columnNum should be set")
+    assert(_rawColumnNum != 0 || _rawLineNum != 0, "lineNum and columnNum should be set")
     if (isInDebugMode) {
       logInfo("Global Pos::" + tokenPos.str + s"::${rawTokens(tokenPos.pos)}")
     }

